const wihtOutEnd = () =>{
  while (true){
    console.log('nunca pares de aprender');
  }
  }


  const fail = (message: string) => {
    throw new Error(message);
  }

  const example = (input: unknown) => {
